app/experimenter/experiments/forms.py

```diff
@@ -1279,10 +1279,51 @@ class NormandyIdForm(ChangeLogMixin, forms.ModelForm):
     normandy_id = forms.IntegerField(
         label="Recipe ID",
         widget=forms.TextInput(
-            attrs={"class": "form-control", "placeholder": "Recipe ID"}
+            attrs={"class": "form-control", "placeholder": "Main Recipe ID"}
         ),
     )
 
+    other_normandy_ids = forms.CharField(
+        label="Other Recipe IDs",
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Other Recipe IDs (Optional)",
+            }
+        ),
+        required=False,
+    )
+
+    def clean(self):
+        # we need to check that both keys exist because if the user enters
+        # a noninteger normandy_id, the key is taken out of cleaned_data
+        # and this method will still be run causing a key error
+
+        if {"normandy_id", "other_normandy_ids"} <= set(self.cleaned_data):
+            if (
+                int(self.cleaned_data["normandy_id"])
+                in self.cleaned_data["other_normandy_ids"]
+            ):
+                raise forms.ValidationError(
+                    {"other_normandy_ids": "Duplicate IDs are not accepted."}
+                )
+
+        return self.cleaned_data
+
+    def clean_other_normandy_ids(self):
+        other_ids_list = []
+
+        if self.cleaned_data["other_normandy_ids"]:
```

Comment:
I don't think you need to do this check, by the time we get to this method the value should always be there:

https://github.com/django/django/blob/master/django/forms/forms.py#L395-L398

Reply:
if this check is not here, the form won't let the other_normandy_ids field be optional.